src/translations/en/publicationTypes.json

```diff
@@ -2,19 +2,28 @@
   "Book": "Book",
   "BookAnthology": "Anthology",
   "BookMonograph": "Monograph",
-  "Chapter": "Chapter",
-  "Degree": "Degree",
-  "DegreeBachelor": "Bachelor",
-  "DegreeMaster": "Master",
-  "DegreePhd": "PhD",
-  "Journal": "Publication in Journal",
+  "ChapterArticle": "Chapter in anthology",
+  "Degree": "Student thesis",
+  "DegreeBachelor": "Bachelor thesis",
+  "DegreeMaster": "Master thesis",
+  "DegreePhd": "Doctoral thesis",
+  "DegreeOther": "Other student thesis",
+  "Journal": "Contribution to Journal",
```

Comment:
synes det er dumt at det er forskjell i stavelse her. Hvorfor er det `Contribution to Journal` (med stor J i journal), mens det er `Chapter in anthology` med liten a? Synes vi skal være konsistente her

Reply:
Hege er på saken